config/setup.py

```diff
@@ -1,6 +1,19 @@
 "This file is here to automatically install the selected DB package"
+import os
+import sys
+from os.path import abspath, dirname
+
```

Comment:
Awesome! didn't knew that that's possible 😅 Or: didn't thought of that solution...
works 👍

Reply:
Thanks. I feel like this solution is a little weird though, but can't think about any other satisfying alternative.